src/store/singing.ts

```diff
@@ -138,20 +137,21 @@ if (window.AudioContext) {
 
 const playheadPosition = new FrequentlyUpdatedState(0);
 const singingVoices = new Map<SingingVoiceSourceHash, SingingVoice>();
-const sequences = new Map<string, Sequence>(); // キーはPhraseKey
+const sequences = new Map<string, Sequence>(); // キーはPhraseSourceHash
```

Comment:
Hashの方は型定義あるので型使えそう？
```suggestion
const sequences = new Map<PhraseSourceHash, Sequence>();
```